edx_toggles/toggles/internal/waffle/legacy.py

```diff
@@ -214,37 +168,32 @@ def __init__(self, waffle_namespace, flag_name, module_name=None):
         self._flag_name = flag_name
         name = f"{waffle_namespace}.{flag_name}"
         super().__init__(name, module_name=module_name, log_prefix=log_prefix)
-        self._set_legacy_custom_attribute()
+        set_custom_attribute(
+            "deprecated_legacy_waffle_class",
+            "{}.{}[{}]".format(
+                self.__class__.__module__, self.__class__.__name__, self.name
+            ),
+        )
 
     @property
     def flag_name(self):
         """
         Non-namespaced flag_name attribute preserved for backward compatibility.
         """
-        set_custom_attribute(
-            "deprecated_waffle_legacy_method",
-            f"WaffleFlag[{self.name}].flag_name",
-        )
         return self._flag_name
```

Comment:
Oops.  I should have been more clear.  Since `deprecated_waffle_legacy_method` is showing me no usage, I think we are ready to delete this whole property.  Same is true for all the other things I noted are not appearing in monitoring.  This clean-up will make it so that the only difference between legacy and non-legacy will be the namespace in the constructor, ensuring the remaining refactors to remove legacy are extremely simple.

Reply:
Right! This explains why you were explicitly listing custom attributes to be removed. I'll remove the methods and add them to the changelog.